.github/actions/chromedriver-updater/chromedriver_updater/__main__.py

```diff
@@ -0,0 +1,172 @@
+#  Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Our PyLint configuration rates this file at 6.9, mostly with trivially fixable things. For some reason it doesn't like the variable name `pr` - says it isn't snake case. That's the kind of thing I tend to ignore, but I believe the contribution guidelines say to keep to 9+